client/src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import toastType from './toastType'
+import { combineReducers } from 'redux'
+
+const allReducers = combineReducers({
```

Comment:
technically you don't need to use the combineReducers helper funciton since you only have one reducer, but this is ok as it is.